frontend/services/entities/global_policies.ts

```diff
@@ -1,7 +1,45 @@
 /* eslint-disable  @typescript-eslint/explicit-module-boundary-types */
+import { snakeCase, reduce } from "lodash";
+
 import sendRequest from "services";
 import endpoints from "utilities/endpoints";
-import { IPolicyFormData, ILoadAllPoliciesResponse } from "interfaces/policy";
+import {
+  IPolicyFormData,
+  ILoadAllPoliciesResponse,
+  IPoliciesCountResponse,
+} from "interfaces/policy";
+import { buildQueryStringFromParams, QueryParams } from "utilities/url";
+
+interface IPoliciesApiParams {
+  page?: number;
+  perPage?: number;
+  orderKey?: string;
+  orderDirection?: "asc" | "desc";
+  query?: string;
+}
+
+export interface IPoliciesQueryKey extends IPoliciesApiParams {
+  scope: "globalPolicies";
+}
+
+export interface IPoliciesCountQueryKey
+  extends Pick<IPoliciesApiParams, "query"> {
+  scope: "policiesCount";
+}
+
+const ORDER_KEY = "name";
+const ORDER_DIRECTION = "asc";
+
+const convertParamsToSnakeCase = (params: IPoliciesApiParams) => {
+  return reduce<typeof params, QueryParams>(
+    params,
+    (result, val, key) => {
+      result[snakeCase(key)] = val;
+      return result;
+    },
+    {}
+  );
+};
```

Comment:
I think this same code is used in a couple of places and can be made a reusable utility.

Reply:
Merging and then can refactor to reusable utility